### PR TITLE
Scenario light

### PR DESCRIPTION
**Pull request by QuanyiLi**
## What changes do you make in this PR?

Scenario LIght support

## Checklist

* [ ] I have merged the latest main branch into current branch.
* [ ] I have run `bash scripts/format.sh` before merging.
* Please use "squash and merge" mode.

### CODE REVIEW

File: metadrive/type.py [hunk 1]
```diff
@@ -125,31 +141,30 @@ def is_traffic_light_in_green(cls, light):
...
-            return [255, 255, 0]
-        if status == self.UNKNOWN:
-            return [180, 180, 180]
+    def simplify_light_status(cls, status: str):
+        """
+        Convert status to red/yellow/green/unknown
+        """
+        if status is cls.LANE_STATE_UNKNOWN or status is cls.LIGHT_UNKNOWN:
+            return cls.LIGHT_UNKNOWN
+        elif status in [cls.LANE_STATE_ARROW_STOP, cls.LANE_STATE_STOP, cls.LANE_STATE_FLASHING_STOP, cls.LIGHT_RED]:
```
Comment (pengzhenghao) [inline metadrive/type.py:162]: Flashing Stop should be counted into UNKNOWN.

Comment (QuanyiLi) [reply metadrive/type.py:163]: IDK, it has a ```stop``` in its name, and I can not find any explanation about this type on internet

Comment (pengzhenghao) [reply metadrive/type.py:163]: Flashing Red light = stop sign in US. Just set it to UNKNOWN.

Comment (QuanyiLi) [reply metadrive/type.py:163]: Done

**Pull request merged**